### PR TITLE
appveyor: Remove clcache

### PR DESCRIPTION
**Pull request by maflcko**
The build time without cache seems to be 47 minutes and with cache 46 minutes. Maybe we can save more time by not installing clcache.

### CODE REVIEW

**Comment by MarcoFalke** [comment]
The build https://ci.appveyor.com/project/DrahtBot/bitcoin/builds/32183943 finished in 44 minutes

Anything left to do here?

File: .appveyor.yml [hunk 1]
```diff
@@ -72,13 +69,9 @@ before_build:
          Write-Host "Qt binaries already present.";
       }
 - cmd: python build_msvc\msvc-autogen.py
-- ps:  Start-Process clcache-server
-- ps:  fsutil behavior set disablelastaccess 0 # Enable Access time feature on Windows (for clcache)
 build_script:
-- cmd: msbuild /p:TrackFileAccess=false /p:CLToolExe=clcache.exe build_msvc\bitcoin.sln /m /v:q /nologo
+- cmd: msbuild /p:TrackFileAccess=false build_msvc\bitcoin.sln /m /v:q /nologo
 after_build:
-- ps:  fsutil behavior set disablelastaccess 1 # Disable Access time feature on Windows (better performance)
```
Comment (sipa) [inline .appveyor.yml:80]: Removing this line seems unrelated.

Comment (MarcoFalke) [reply .appveyor.yml:80]: See this build here: https://ci.appveyor.com/project/DrahtBot/bitcoin/builds/32194931

So with this line added, it slowed down the build. Or at least didn't speed it up, so I don't feel comfortable keeping it.

Test code should be minimal, easy to read and easy to write. Having a line say "better performance" without any data to back up the claim doesn't help.

**Comment by MarcoFalke** [comment]
I am going to go ahead and merge this. If someone finds a way to speed things up, great, we can take them. Though, it doesn't look like clcache speeds things up at all for us. This might be a configuration error or it might be vaporvare, and keeping in the code in the appveyor yaml isn't helping anyone.
